mne/viz/tests/test_raw.py

```diff
@@ -827,6 +827,87 @@ def test_remove_annotations(raw, hide_which, browser_backend):
     assert len(raw.annotations) == len(hide_which)
 
 
+def test_merge_annotations(raw, browser_backend):
+    """Test merging of annotations in the Qt backend.
+
+    Let's not bother in figuring out on which sample the _fake_click actually
+    dropped the annotation, especially with the 600.614 Hz weird sampling rate.
+    -> atol = 10 / raw.info["sfreq"]
```

Comment:
If you have a better idea to get that test working, I'm listening. In https://github.com/mne-tools/mne-qt-browser/pull/178, it failed the Ubuntu and Windows CIs ([link](https://github.com/mne-tools/mne-qt-browser/actions/runs/5751433788/job/15590173072)) with `atol = 2 / raw.info["sfreq"]`. Locally, on an ubuntu-based distro it was passing..

It looks to me like the `_fake_click` location is not very precise. I tried changing the onset/duration/click locations with values corresponding to an exact sample, without luck. For instance, I tried to click on the right edge of an annotation and it was working.. within +/- 15 samples of the actual edge location.

Reply:
> It looks to me like the _fake_click location is not very precise

Yes it's possible and I think it's okay to have a loose tolerance. As long as it's strict enough to catch if we broke things it should be fine. We can always improve the tolerance later